server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1861,8 +1861,10 @@ private CompactionStats _majorCompact(MajorCompactionReason reason) throws IOExc
       if (inputFiles.isEmpty()) {
         if (reason == MajorCompactionReason.USER) {
           // no work to do
-          lastCompactID = compactionId.getFirst();
-          updateCompactionID = true;
+//          lastCompactID = compactionId.getFirst();
```

Comment:
Yes, touching `lastCompactID` is sensitive because Accumulo uses that to determine when a compaction has already occurred. I think the modifications are correct, but I retained the commented lines because I feel unsure.  I will remove the commented lines and try to test further.